example/example.js

```diff
@@ -9,8 +9,16 @@ const throttledEmbaralha = (elem, time) => {
 const main = () => {
   const body = document.querySelector('body');
   for (const elem of body.childNodes) {
-    elem.addEventListener('mouseover', throttledEmbaralha(elem, 1000));
+    // Added a timeout to stop it from executing multiple times at once,
+    // Note the timeout is 50 more than initial because it might go over when both are 1000
+    setTimeout(() => {elem.addEventListener('mouseover', throttledEmbaralha(elem, 1000))}, 1050);
```

Comment:
This isn't solving the issue, it's just creating more event listeners with the same issue. You can test it locally with `npm start` and then going to `http://localhost:8080` in your browser and see it